test/curve_analysis/test_curve_fit.py

```diff
@@ -63,6 +64,67 @@ class TestAnalysis(CurveAnalysis):
     return TestAnalysis()
 
 
+class TestFitData(QiskitTestCase):
+    """Unittest for fit data dataclass."""
+
+    def test_get_value(self):
+        """Get fit value from fit data object."""
+        data = FitData(
+            popt=np.asarray([1.0, 2.0, 3.0]),
+            popt_keys=["a", "b", "c"],
+            popt_err=np.asarray([0.1, 0.2, 0.3]),
+            pcov=np.diag(np.ones(3)),
+            reduced_chisq=0.0,
+            dof=0,
+            x_range=(0, 0),
+            y_range=(0, 0),
+        )
+
+        a_val = data.fitval("a")
+        self.assertEqual(a_val, FitVal(1.0, 0.1))
+
+        a_val = data.fitval("b")
+        self.assertEqual(a_val, FitVal(2.0, 0.2))
+
+        a_val = data.fitval("c")
+        self.assertEqual(a_val, FitVal(3.0, 0.3))
```

Comment:
I guess that you wanted to call the variables `b_val` and `c_val`

Reply:
Good catch!